python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -91,7 +91,7 @@ class RandomForestClassifier(BaseRandomForestModel, DelayedPredictionMixin,
     bootstrap_features : boolean (default = False)
         Control bootstrapping for features.
         If features are drawn with or without replacement
-    rows_sample : float (default = 1.0)
+    max_samples : float (default = 1.0)
```

Comment:
We should deprecate the old parameter for one release since it has been public. Set the old parameter to None. If it's not None, use it but issue a deprecation warning. No need to keep the old parameter documented (just code), so I think you can make that change in one place.

Reply:
Done